kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -254,13 +270,6 @@ define([
                             collapsedCellJobStatus,
                             // options dropdown
                             renderOptions(_cell, events),
-                            // what are these buttons for?
```

Comment:
removed unused buttons

Reply:
I don't remember seeing those any time recently, and there's a different triangle button for showing that an app has a new version. I can't tell what would make those not hidden. Maybe those should appear if the app is running?

Reply:
But with some manual tinkering and forcing them to show, they look suspiciously out of place. I think they're fine to remove.